testing/liberasurecode/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Ehud Kaldor <ehud@unfairfunction.org>
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="COPYING" 
+depends=""
+makedepends="libtool make automake autoconf gcc musl-dev zlib-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}-src.tar.gz::https://github.com/openstack/liberasurecode/archive/1.5.0.tar.gz"
```

Comment:
Please use `$pkgver` which makes it easier to maintain it (upgrade means simply change `pkgver`)
````sh
source="${pkgname}-${pkgver}-src.tar.gz::https://github.com/openstack/liberasurecode/archive/$pkgver.tar.gz"
````

Reply:
done, I think...